src/app.js

```diff
@@ -16,6 +17,7 @@ const App = () => (
     <Home path="/">
       <Processors path="processors">
         <ProcessorsHome path="/" />
```

Comment:
Not needed there.
Try to move it to `Processors` page and check if `this.props.chidren` empty

https://reactjs.org/docs/react-api.html#reactchildrencount

if empty - render `ProcessorsHome` otherwise - `this.props.children`